src/js/languages/default.json

```diff
@@ -48,5 +48,9 @@
     "playButton": "play",
     "volumeDown": "volume down",
     "volumeUp": "volume up"
+  },
+  "dateInput": {
+    "enterCalendar": "Enter calendar dialog",
```

Comment:
Should we do something slightly more descriptive like the one above for textinput. Maybe "Calendar is open, press tab to enter the calendar."

I feel like that might create better context for someone using a screen reader.

Reply:
@taysea yeah, I think that makes more sense.